examples/scroll/lv_example_scroll_1.c

```diff
@@ -1,5 +1,77 @@
 #include "../lv_examples.h"
 #if LV_BUILD_EXAMPLES
+#include <stdio.h>  /* For sprintf() prototype. */
```

Comment:
Note that the examples are shown in 320x240 resolution on the website. 
![image](https://github.com/user-attachments/assets/bd164d39-77e6-4eb4-aa0f-272a1eadd08c)

I like the idea to add more info, but I suggest changing the labels to logs.

Reply:
> I like the idea to add more info, but I suggest changing the labels to logs.

Darn, I was hoping that would come out nicer....

I like the idea of logging as well, but any level above LV_LOG_INFO() seems too high, and -- from my own experience -- I would never turn on LV_LOG_INFO() because there was so much info that I feared whatever I was looking for would get lost.  (I was using LVGL v8.4.)  Is LV_LOG_INFO() what you were thinking?  (I did not explore it thoroughly a year ago, so I may not be aware of how useful it is, although I did find LV_LOG_WARN() and LV_LOG_ERROR() *very* useful!)

Let me know -- I'm happy to change it to make it helpful for others.

Reply:
FYI, I won't be hurt if you decide we should just skip changing the example.  I found it very useful to understand the functions, so I thought others would as well.

Reply:
I think its very useful and and we should keep it these info. This way the example shows the `lv_scroll_get_*` functions which as great!

`LV_LOG_USER` (highest level) seems like a good choice. I'd add a single multi-line log for better readability.